src/lib/utils/theme.utils.ts

```diff
@@ -0,0 +1,56 @@
+import { Theme } from "../types/theme";
+import { enumFromStringExists } from "./enum.utils";
+
+const isNode = (): boolean =>
```

Comment:
Is there a plan to include `@dfinity/ic-js/utilis` into `gix-component`?

Reply:
I'm not sure, `utils` has a peer dependencies on `agent-js` which is not use here. Let's discuss it!